docs/client-sdk-states-and-lifecycle.mdx

```diff
@@ -0,0 +1,414 @@
+---
+id: client-sdk-states-and-lifecycle
+title: Client SDK States & Lifecycle
+slug: /client-sdk-states-and-lifecycle
+---
+
+Nimbus SDK Experiment States and Lifecycle
+
+
+###### Authors: Ryan Kelly
+###### Reviewers: Kate Hudson, Tim Smith
```

Comment:
Question from @rfk from the original doc:

@k88hudson @tdsmith did you have any further feedback on the details here, particularly around the circumstances under which we emit different metrics?